embed/etcd.go

```diff
@@ -219,23 +219,30 @@ func (e *Etcd) Config() Config {
 	return e.cfg
 }
 
+// Close gracefully shuts down all servers/listeners.
```

Comment:
we should mention there is a default timeout for the graceful shutdown. or we will do a force shutdown.

Reply:
@xiang90 Comments are added. PTAL.